### PR TITLE
#1159 initial user wbs page

### PR DESCRIPTION
**Pull request by krzyk**
#1159 
* created initial "User WBS" page - current one is a mock displaying only a list of user project pids
* added two todos to continue implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1628](https://github.com/zerocracy/farm/issues/1628) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1628](https://github.com/zerocracy/farm/issues/1628) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2340532037); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        people.invite(uid, "yegor256");
+        new Projects(farm, uid).bootstrap().add(project.pid());
+        MatcherAssert.assertThat(
+            new RsPrint(
+                new TkApp(farm).act(
+                    new RqWithUser(
+                        farm,
+                        new RqFake(
+                            new ListOf<>(
+                                String.format("GET /u/%s/wbs", uid),
```
Comment (paulodamaso) [inline src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @krzyk Let's use `FormattedText` here.

File: .rultor.yml [hunk 2]
```diff
@@ -41,7 +41,8 @@ release:
     git fetch
     git add settings.xml
     mvn initialize -Pshutdown --settings ../settings.xml
-    git commit -m 'settings.xml' && git push -f heroku $(git symbolic-ref --short HEAD):master && git reset HEAD~1
+    git commit -m 'settings.xml'; trap "git reset HEAD~1" EXIT
```
Comment (paulodamaso) [inline .rultor.yml]: @krzyk Why these changes?

File: src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * Test case for {@link TkUserWbs}.
+ * @since 1.0
+ * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+public final class TkUserWbsTest {
+
+    @Test
+    public void rendersUserProjectPid() throws Exception {
```
Comment (paulodamaso) [inline src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @krzyk I didn't get what is the purpose of this test; how it should help us to assert that `WBS User` page is being displayed correctly?

File: .rultor.yml (hunk 2 above)
Comment (krzyk) [reply .rultor.yml]: @paulodamaso It is from merge of master, not sure why it showed up here

File: src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java (hunk 3 above)
Comment (krzyk) [reply src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @paulodamaso User WBS page will contain a list of user projects ids (we will list project pid + job id there), right now we show only pids, so it tests that.

**Comment by krzyk** [comment]
@paulodamaso updated, please check

File: src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java (hunk 3 above)
Comment (paulodamaso) [reply src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @krzyk I think that it is too little to test that the page is being correctly assembled, maybe you should put some pieces of the page code to see if it is right.

Comment (krzyk) [reply src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @paulodamaso but the page is very simple, just a list of PIDs, there is no more to test there, this is literally the full page right now.

Comment (paulodamaso) [reply src/test/java/com/zerocracy/tk/profile/TkUserWbsTest.java]: @krzyk Okay then

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1628](https://github.com/zerocracy/farm/issues/1628) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@paulodamaso According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by paulodamaso** [comment]
@ypshenychka sorry, what issue reported was cosmetic?

**Comment by ypshenychka** [comment]
@paulodamaso I'm not saying about cosmetic issues. Just saying that you found only 2 issues: [this](https://github.com/zerocracy/farm/pull/1628#discussion_r212058072) and [this](https://github.com/zerocracy/farm/pull/1628#discussion_r212058180)

**Comment by paulodamaso** [comment]
@ypshenychka right, thanks. I confirm that I'll try to find 3 or more next time.

**Comment by ypshenychka** [comment]
@paulodamaso thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
